### ISSUE TITLE
MessageBar: [a11y] Doesn't announce the message on show

### ISSUE THREAD

**Issue by mdahamiwal** [OfficeDev/office-ui-fabric-react#866]
### Bug Report

- __Package version(s)__: master
- __Browser and OS versions__: all

#### Describe the issue:

When messagebar is added to a page, the AT doesn't announce the message even though message has an internal live region.

#### Actual behavior:
Message text doesn't get announced by ATs

#### Expected behavior:
Message should be announced by ATs

### If applicable, please provide a codepen repro:
http://codepen.io/mdahamiwal/pen/bgWJbx

**Comment by mdahamiwal** [comment]
ATs track live region text updates and doesn't announce the content at the time of creation. In messageBar we create a live region with Error/Info text but since its a one time content ATs will not announce the update.

Fix:
Update the MessageBar text content with some delay after the live region is created.
http://codepen.io/mdahamiwal/pen/XpROQP
